packages/ember-toucan-core/src/components/form/fields/radio.hbs

```diff
@@ -17,14 +17,43 @@
         />
 
         <div class="flex flex-col">
-          <span
-            class="type-md-tight block leading-4
-              {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
-            data-label
-          >{{@label}}</span>
 
-          {{#if @hint}}
-            <field.Hint data-hint>{{@hint}}</field.Hint>
+          {{#if
+            (this.assertBlockOrArgumentExists
+              (hash
+                blockExists=(has-block "label")
+                argName="label"
+                arg=@label
+                isRequired=true
+              )
+            )
+          }}
+            <span
+              class="type-md-tight block leading-4
+                {{if @isDisabled 'text-disabled' 'text-titles-and-attributes'}}"
+              data-label
+            >
+              {{#if (has-block "label")}}
+                {{yield to="label"}}
+              {{else}}
+                {{@label}}
+              {{/if}}
+            </span>
+
```

Comment:
superrrr nit: looks like some extra spacing here that's not in the other files.

Reply:
The extra line space should be removed now...I notice that this isn't fixed with `pnpm format` which is a bit of a bummer.